com.github.rafostar.Clapper.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "com.github.rafostar.Clapper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.rafostar.Clapper",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-videos",
```

Comment:
Is it actually needed for anything? `GtkFileChooserNative` uses portals by default.

Reply:
I described what all these permissions do in the "submission criteria" checklist (first comment).

Reply:
This one is for video playlist from text file. Basically user can write a text file with filepaths to videos (one per line) and open it as a video playlist in the app. I do not want to use `host` here, so I am limiting this functionality to videos inside user home videos directory for flatpak version.